.github/workflows/docker-image-build.yml

```diff
@@ -8,14 +8,14 @@ on:
       - 'doc/ctsm-docs_container/**'
```

Comment:
I think we might want to have a 

``` yaml
branches:
   - master
   - b4b-dev
```

Above this so it only runs on push to master and b4b-dev? Which would mean it wouldn't run when tags are made, which is probably what we want, because the tag almost always will be identical to the PR merged to master. I wasn't sure we want it on b4b-dev either, but because changes like this will go on b4b-dev, you probably do want it to run.

Reply:
I think it makes sense for this workflow to run whenever the listed files are changed, regardless of what branch the changes are on. Remember, this is not the **publish** workflow; this only checks that the image can be built. Also note that, because this workflow runs on every pull request, the "run on push" only ever comes into play when someone pushes changes in those files **to a branch without doing a PR**.

The "run when tags are made" behavior will be addressed in a forthcoming PR.